src/py42/sdk/queries/fileevents/filters/exposure_filter.py

```diff
@@ -124,6 +124,14 @@ def choices():
         return get_attribute_keys_from_class(SyncDestination)
 
 
+class SyncDestinationUsername(FileEventFilterStringField):
+    """Class that filters events based on the name of the username associated with the cloud service
```

Comment:
probably don't need to say `name of the username`, and could just say `username`